stubs/common/Facades.stub

```diff
@@ -5,6 +5,7 @@ namespace Illuminate\Support\Facades;
 abstract class Facade {}
 
 /**
+ * @mixin \Illuminate\Redis\Connections\Connection
  * @mixin \Redis
```

Comment:
```suggestion
 * @mixin \Redis
 * @mixin \Illuminate\Redis\Connections\Connection
```

Reply:
The connection needed to be before `\Redis` so that the `client()` method signature on the connection class takes priority